<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>610</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>30</number>
     </property>
     <property name="verticalSpacing">
      <number>8</number>
     </property>
     <item row="0" column="0">
      <widget class="QTabWidget" name="option_tab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>520</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>520</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="main_opts">
        <attribute name="title">
         <string>Main Options</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="file_opts">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <attribute name="title">
         <string>File Options</string>
        </attribute>
        <widget class="QPlainTextEdit" name="dir_path">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>data is stored in directory DIR</string>
         </property>
         <property name="plainText">
          <string>./</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>59</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>Data Dir</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>59</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>Psifile</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="psi_path">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;the wavefunction is read FILE rather than from ./psi. The string FILE may be a relative or a full path-name &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="plainText">
          <string/>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="pes_path">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;the PES is read from FILE rather than from ./oper. The string FILE may be a relative or a full path-name.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="plainText">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>59</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>PES file</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="out_path">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>440</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>The ouput is written to file FILE.pl rather than to ./surface.pl \n                            The string FILE may be a relative or a full path-name.</string>
         </property>
         <property name="plainText">
          <string>surface </string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>420</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Output File</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>370</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Output Dir</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="out_dir">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>390</y>
           <width>151</width>
           <height>31</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>The output is written to directory DIR. \n                            This is useful, \n                            if one has no right to write to the name directory. \n                            The directory DIR must exist. \n                            The string DIR may be a relative or a full path-name.</string>
         </property>
         <property name="plainText">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="flag_tab">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="locale">
         <locale language="C" country="AnyCountry"/>
        </property>
        <attribute name="title">
         <string>Other Options</string>
        </attribute>
        <widget class="QCheckBox" name="lshowpsi">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>85</width>
           <height>21</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Only the OPER file is read, i.e. no wavefunction plotting.</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>lshowpsi</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="lnopes">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>85</width>
           <height>21</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>The OPER file is not read, no potential plotting, no dia-&gt;ad trafo.</string>
         </property>
         <property name="text">
          <string>lnopes</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="lrediab">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>85</width>
           <height>21</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>The raw adiabatic PES from a DD,calculation is plotted.</string>
         </property>
         <property name="text">
          <string>lrediab</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="ldblocal">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>85</width>
           <height>21</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>The local rather than interpolated energy is plotted.</string>
         </property>
         <property name="locale">
          <locale language="C" country="AnyCountry"/>
         </property>
         <property name="text">
          <string>ldblocal</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="lshowrst">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>150</y>
           <width>85</width>
           <height>21</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>The restart file is read, rather than the psi file.</string>
         </property>
         <property name="locale">
          <locale language="C" country="AnyCountry"/>
         </property>
         <property name="text">
          <string>lshowrst</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="lnw">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>180</y>
           <width>85</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>lnw</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="lpop2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>220</y>
           <width>85</width>
           <height>21</height>
          </rect>
         </property>
         <property name="locale">
          <locale language="C" country="AnyCountry"/>
         </property>
         <property name="text">
          <string>lpop2</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="lpop2all">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>260</y>
           <width>85</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>lpop2all</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="ncut">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>40</y>
           <width>91</width>
           <height>24</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Only the first num WFs are read from psi-file.</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>10000000</number>
         </property>
         <property name="value">
          <number>999999</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>20</y>
           <width>59</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>ncut</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="nskip">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>90</y>
           <width>91</width>
           <height>24</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>The first m WFs are skipped</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>10000000</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>70</y>
           <width>59</width>
           <height>15</height>
          </rect>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>nskip</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="step">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>140</y>
           <width>91</width>
           <height>24</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Only every step's WF will be processed.</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>10000000</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>120</y>
           <width>59</width>
           <height>15</height>
          </rect>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>step</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="graph_canvas">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>601</width>
         <height>491</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>601</width>
         <height>491</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>test1_01.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextBrowser" name="info_txt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>601</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>601</width>
         <height>61</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="run_button">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_FIle"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen_FIle">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
